src/lib/components/onboarding/scripts.ts

```diff
@@ -0,0 +1,105 @@
+import type { DriveStep } from "driver.js";
+
+export const scripts: Record<string, DriveStep[]> = {
```

Comment:
Any ideas for how to do i18n for these scripts effectively? Should the title and description for each step be in `en.json`?

Reply:
Probably should be in `en.json`. I don't think we've found a great pattern for doing i18n outside of components.